### PR TITLE
[ENH] Usage of nipype MapNode

### PR DESCRIPTION
**Pull request by pdedumast**

### CODE REVIEW

**Comment by pdedumast** [comment]
@sebastientourbier seems to be good now 👌

File: pymialsrtk/pipelines/anatomical/srr.py [hunk 1]
```diff
@@ -485,14 +506,16 @@ def create_workflow(self):
...
-        self.wf.connect(srtkMaskImage01, ("output_images", utils.sort_ascending), datasink, 'anat.@LRsDenoised')
+        self.wf.connect(srtkMaskImage01, ("out_im_file", utils.sort_ascending), datasink, 'anat.@LRsDenoised')
         self.wf.connect(srtkImageReconstruction, ("output_transforms", utils.sort_ascending), datasink, 'xfm.@transforms')
 
         self.wf.connect(srtkImageReconstruction, "output_sdi", datasink, 'anat.@SDI')
         self.wf.connect(srtkN4BiasFieldCorrection, "output_image", datasink, 'anat.@SR')
         self.wf.connect(srtkTVSuperResolution, "output_json_path", datasink, 'anat.@SRjson')
         self.wf.connect(srtkHRMask, "output_srmask", datasink, 'anat.@SRmask')
 
+        return
```
Comment (sebastientourbier) [inline pymialsrtk/pipelines/anatomical/srr.py:517]: What is this return?

**Review by sebastientourbier — APPROVED** [review]
Another great job @pdedumast! 
After my comment about the  "return" is resolved,  I will merge 🎉

File: pymialsrtk/pipelines/anatomical/srr.py (hunk 1 above)
Comment (pdedumast) [reply pymialsrtk/pipelines/anatomical/srr.py:517]: Used for debugging during the MapNode refactoring... :see_no_evil:

**Pull request merged**